app/test_main.py

```diff
@@ -1,3 +1,19 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "example_input, expected",
+    [
+        ((0, 0), [0, 0]),
+        ((14, 14), [0, 0]),
+        ((15, 15), [1, 1]),
+        ((23, 23), [1, 1]),
+        ((24, 24), [2, 2]),
+        ((27, 27), [2, 2]),
+        ((28, 28), [3, 2]),
+        ((100, 100), [21, 17])
+    ]
+)
+def test_get_human_age(example_input: tuple, expected: list) -> None:
+    assert get_human_age(example_input[0], example_input[1]) == expected
```

Comment:
Better, but still not clear enough. You can provide those `example_input` As two different variables and named them informative